test/ArgumentForwardingTests/ArgumentForwardingTests.cs

```diff
@@ -51,6 +51,7 @@ private void FindAndEnsureReflectorPresent()
         /// <param name="testUserArgument"></param>
         [Theory]
         [InlineData(@"""abc"" d e")]
+        [InlineData(@"""ábc"" d é")]
```

Comment:
I'm not sure I follow this fully.  Can you explain a little more what you'd like to see?  What would the fact do that this doesn't?

Also, the `dotnet run` tests exercise this in a more end-to-end manner since dotnet run will spawn the app process, and forward the output back to the CLI.